### PR TITLE
Add documentation for the housekeeping

### PR DESCRIPTION
**Pull request by paraita**

### CODE REVIEW

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 1]
```diff
@@ -1462,6 +1462,25 @@ that automatically adds indexes on foreign keys, it is strongly recommended to
...
+==== Housekeeping
+
+The Scheduler provides a housekeeping mechanism that periodically removes finished jobs
+from the Scheduler Portal. It can also remove jobs and all its data from the database to save space.
+This mechanism has two phases:
+
+. Once a job is finished, set its scheduled time for removal (the job expiration date)
+. Periodically trigger the cleaning of the expired jobs off the scheduler and/or the database
+
+One can edit the `pa.scheduler.core.automaticremovejobdelay` parameter to set the delay before
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: add which configuration file should be edited

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 2]
```diff
@@ -1462,6 +1462,25 @@ that automatically adds indexes on foreign keys, it is strongly recommended to
 ask your DBA to identify and delete duplicate indexes created by the database
 system since they may hurt performances.
 
+==== Housekeeping
+
+The Scheduler provides a housekeeping mechanism that periodically removes finished jobs
+from the Scheduler Portal. It can also remove jobs and all its data from the database to save space.
+This mechanism has two phases:
+
+. Once a job is finished, set its scheduled time for removal (the job expiration date)
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: maybe use a numbered list instead of unnumbered

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 3]
```diff
@@ -1462,6 +1462,25 @@ that automatically adds indexes on foreign keys, it is strongly recommended to
...
+. Periodically trigger the cleaning of the expired jobs off the scheduler and/or the database
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: I would rephrase for better understanding. "Actual cleaning of expired jobs from the scheduler and/or the database will be periodically triggered and performed in a bulk operation."

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 2 above)
Comment (paraita) [inline src/docs/admin/ProActiveAdminGuide.adoc]: thanks for the comments ! @fviale the numbers are generated automatically.

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 1 above)
Comment (paraita) [inline src/docs/admin/ProActiveAdminGuide.adoc]: yes ! thanks for pointing that out

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 3 above)
Comment (paraita) [inline src/docs/admin/ProActiveAdminGuide.adoc]: yes your sentence is more understandable I changed it !

**Comment by tobwiens** [comment]
:+1:
